GiveNow/GiveNow/Info.plist

```diff
@@ -22,6 +22,14 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>MGLMapboxAccessToken</key>
+	<string>sk.eyJ1Ijoic2F1bG1vcmEiLCJhIjoiY2lpOXN3cDZ4MDBjeHR2bHphNDEwY2I5dyJ9.ITbGw4wQ8UPu9D5Kfu1qRg</string>
```

Comment:
Yeah, I expect this key to go there too. I just didn't want to forget it and not have mapbox work for everyone. 

Sent from my iPhone

> On Dec 16, 2015, at 11:10 PM, Brennan Stehling <notifications@github.com> wrote:
> 
> In GiveNow/GiveNow/Info.plist:
> 
> > @@ -22,6 +22,14 @@
> >  	<string>1</string>
> >  	<key>LSRequiresIPhoneOS</key>
> >  	<true/>
> > +	<key>MGLMapboxAccessToken</key>
> > +	<string>sk.eyJ1Ijoic2F1bG1vcmEiLCJhIjoiY2lpOXN3cDZ4MDBjeHR2bHphNDEwY2I5dyJ9.ITbGw4wQ8UPu9D5Kfu1qRg</string>
> Once I place the private keys in keys.plist we can put this file there since Mapbox does allow this key to be set with code so we do not have to put it in Info.plist.
> 
> —
> Reply to this email directly or view it on GitHub.
>